omc3/tune_analysis/kick_file_modifiers.py

```diff
@@ -132,14 +128,13 @@ def _filter_bbq_cut(bbq_df: tfs.TfsDataFrame, filter_opts: Sequence[MinMaxFilter
     bbq_df.headers[get_fine_cut_header()] = filter_opts[0].fine_cut
 
     for idx, plane in enumerate(PLANES):
-        bbq_mav, bbq_std, mask = bbq_tools.get_moving_average(bbq_df[get_bbq_col(plane)], filter_opts[idx])
+        bbq_mav, bbq_err, mask = bbq_tools.get_moving_average(bbq_df[get_bbq_col(plane)], filter_opts[idx])
 
         bbq_df.headers[get_min_tune_header(plane)] = filter_opts[idx].min
         bbq_df.headers[get_max_tune_header(plane)] = filter_opts[idx].max
 
         bbq_df[get_mav_col(plane)] = bbq_mav
-        # bbq_df[get_mav_err_col(plane)] = bbq_std  # this is too large
-        bbq_df[get_mav_err_col(plane)] = 0.  # TODO to be discussed with Ewen and Tobias (jdilly, 2022-05-23)
+        bbq_df[get_mav_err_col(plane)] = bbq_err  # TODO to be discussed with Ewen and Tobias (jdilly, 2022-05-23)
```

Comment:
Out of curiosity, what justifies including this when previously it was commented out since "this is too large"?

Reply:
I always wanted to include it here, but the way bbq_std was calculated was overestimating the error, as it was a standard std() over the BBQ-window. Now it is still the std() over the window, but with the moving average subtracted, i.e. the deviation of the datapoints not to their mean, but to the sliding average.

Reply:
Ah I had not caught that on my read, sounds good to me 👍